game-app/game-core/src/main/java/tools/image/PolygonGrabber.java

```diff
@@ -77,13 +76,13 @@ private void runInternal() throws IOException {
     mapFolderLocation = MapFolderLocationSystemProperty.read();
     log.info("Select the map");
     final FileOpen mapSelection = new FileOpen("Select The Map", mapFolderLocation, ".gif", ".png");
-    final String mapName = mapSelection.getPathString();
+    final Path mapName = mapSelection.getFile();
```

Comment:
Ouch,this legacy code is mostly incorrect here to assume that 'mapName' is the folder name  : (
I'm not sure how to fix it or if it matters, this perhaps could be "fixed" simply by renaming variables as this is no longer the map name but is just the map folder.

Reply:
Yeah true, the map making tools are mostly technical dept at this point it really feels unsatisfying to fix those

Reply:
Agree, this update we can let slide I suppose. For a more generic util that might get re-used, in that case we should resolve the concern.